base/doc/ltnews40.tex

```diff
@@ -155,88 +156,198 @@
+\pdfTeX{} continues to be supported for existing material, but for new
+documents, moving to \LuaTeX{} is recommended.
 
 We cannot make statements about the support for other engines such as
-(u)p\TeX{}, as we don't use these programs nor have in depth knowledge of their
-functionalities. To the best of our knowledge, core \LaTeX{} works well with
-these engines, but if and to what extent tagging can be supported will remain
-to be seen. If relevant information becomes available to us we will provide an
-update in future editions of the \LaTeX{} newsletter.
+(u)p\TeX{}, as we don't use these programs nor have in depth knowledge
+of their functionalities. To the best of our knowledge, core \LaTeX{}
+works well with these engines, but if and to what extent tagging can
+be supported will remain to be seen. If relevant information becomes
+available to us we will provide an update in future editions of the
+\LaTeX{} newsletter.
 
-\section{Switch to T1 as default encoding in documents using \cs{DocumentMetadata}}
 
-As it is well known the font encoding \texttt{OT1} supports only 128 characters and 
-has various problems and quirks notably for languages different to English. 
-Nevertheless \texttt{OT1} is the default encoding in \LaTeX{} and this can not be easily 
-changed without affecting many documents as the \texttt{T1} version of the fonts have slightly different metrics. 
+\subsection{Tagging support for external packages}
 
-The introduction of the \cs{DocumentMetadata} command, which announces 
-\emph{new} code and changes that can also affect the layout gives us now the 
-opportunity to make this step. So with this version a use of 
-\cs{DocumentMetadata} with (pdf)\LaTeX{} will setup  \texttt{T1} as default 
-font encoding\footnote{The Unicode engines will continue to use \texttt{TU} 
-as encoding.}. To ensure that scalable fonts are used, the package 
-\pkg{cm-super} has to be installed. Users who want to revert to the 
-\texttt{OT1} encoding in their document can do so with
-\verb+\usepackage[OT1]{fontenc}+. 
+At \url{https://latex3.github.io/tagging-project/tagging-status/} we
+show the status of many \LaTeX{} Packages and Classes with respect to
+PDF tagging.  We also started to improve tagging support in external
+packages. If the \texttt{firstaid} key is used in addition to the
+\texttt{phase-III} key basic commands of packages including
+\pkg{amsthm} and \pkg{fancyvrb} can now be used.
 
 
-
-\section{News from the \enquote{\LaTeX{} Tagged PDF} project}
+\subsection{Improved table tagging}
 
 The tagging of tabulars has been extended: it is now possible to tag
-also row headers and to create cells that span more than one row. 
+row headers and to create cells that span more than one row.
+
+The interface to this functionality is not finalized but can be
+accessed in the current release by specifying the row and columns to
+be treated as headers.
+
+\begin{verbatim}
+\tagpdfsetup{
```

Comment:
Reminds me I want to talk generally about setting keys - one for a team meeting!